README.md

```diff
@@ -4,6 +4,7 @@ The main credentials, **databaseURL** & the **storageBucket** are automatically
   
 ```sh
 # each arg must have at least 2-part key (e.g foo.bar)
+$ firebase functions:config:set website.base_url="Base url of the website"
```

Comment:
I found an article that says the hosting URL is formatted like this
`<project-name>.firebaseapp.com/`
and the function url is like
`https://us-central1-<project-name>.cloudfunctions.net/<function-name>`

the function url can be obtained from the request header **`host`**, So yes ... it can be found easily.

Reply:
The hosting will be deployed to a custom domain, so I guess this is not a way to go.

For now let's get the domain from config. However use `frontend.domain` config variable to get the domain only, and construct URL from it.